Sources/Flows/OAuth2.swift

```diff
@@ -196,7 +194,7 @@ open class OAuth2: OAuth2Base {
 				}
 				else {
 					if let err = error {
-						self.logger?.debug("OAuth2", msg: "\(err)")
+                        self.logger?.debug("OAuth2", msg: "Error refreshing token: \(err)")
```

Comment:
Use tabs instead of spaces

Reply:
Ug, sorry, Xcode defaults kicking in. In our internal projects, we've got [swift format](https://github.com/nicklockwood/SwiftFormat) as a build phase and I get used to not checking formatting :)